Dockerfile

```diff
@@ -1,11 +1,32 @@
-FROM node:16.13.1-alpine3.15
+# syntax=docker/dockerfile:1
+FROM node:lts-alpine3.17 as builder
 
-WORKDIR /usr/src/app
+WORKDIR /app
 
 COPY package*.json ./
+COPY prisma ./prisma
+
 RUN npm install
 
 COPY . .
-RUN npx prisma generate && npx prisma migrate dev --name init && npm run build
 
-CMD ["npm", "run", "serve"]
+RUN npm run build && npx prisma generate
+
+FROM node:lts-alpine3.17
+
+WORKDIR /app
+
+RUN addgroup -S twiggy && adduser -S twiggy -G twiggy
+
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/assets ./assets
+COPY --from=builder /app/package*.json ./
+COPY --from=builder /app/build ./build
+COPY --from=builder /app/prisma ./prisma
+COPY --from=builder /app/prisma/database.empty ./prisma/db/main.db
```

Comment:
I'm not a fan of doing this type of renames instead of having the same filename inside and outside, so I would suggest having the same name.

Reply:
However since this is for a production Dockerfile only for now is not as bad.